ecl/hql/hqlexpr.cpp

```diff
@@ -2002,6 +2002,7 @@ const char *getOpString(node_operator op)
     case no_selectindirect: return ".<>";
     case no_isomitted: return "no_isomitted";
     case no_getenv: return "GETENV";
+    case no_getsecretvalue: return "GETSECRETVALUE";
```

Comment:
Right, I actually had it as GETSECRET, but changed it since it's accessing a key within a secret and people already jumble up the terminology.  But I agree GETSECRET is cleaner and more intuitive.